components/programmes/ConditionsOfJoining.tsx

```diff
@@ -40,6 +40,20 @@ export function ConditionsOfJoining({
           {CojUtilities.getVersionText(conditionsOfJoining.version)}
         </SummaryList.Value>
       </SummaryList.Row>
+      <SummaryList.Row>
+        <SummaryList.Value

```

Comment:
This really should be a button within a `SummaryList.Actions` to stay consistent with the "Sign" button.